CMakeLists.txt

```diff
@@ -396,11 +396,11 @@ endif ()
 # ----------------------------------------------------------------------------
 # installation directories
 if (OS_WINDOWS)
-  set (RUNTIME_INSTALL_DIR Bin)
-  set (LIBRARY_INSTALL_DIR Lib)
-  set (INCLUDE_INSTALL_DIR Include)
-  set (CONFIG_INSTALL_DIR  CMake)
-  set (PKGCONFIG_INSTALL_DIR)
+  set (RUNTIME_INSTALL_DIR "bin")
```

Comment:
Ok, just in general cmake projects on Windows use the lowercase naming convention (almost all from https://github.com/Microsoft/vcpkg/tree/master/ports).